content/post/council-meeting-2024-04-01/index.md

```diff
@@ -0,0 +1,42 @@
+
+There is one ordinance second reading on the docket tonight. [PH-1/B-1](https://a2gov.legistar.com/LegislationDetail.aspx?ID=6551472&GUID=4D77DEE9-94F7-42DA-B405-3018DC6F9078&Options=&Search=) is the second reading of changes to the Open Space and Parkland Preservation ordinance AKA the Greenbelt. This would, among other things, include some areas of Ypsilanti Township within the Greenbelt and also change some of the rules governing conservation easements.
+
+There are two ordinance first readings tonight. [C-1](https://a2gov.legistar.com/LegislationDetail.aspx?ID=6586150&GUID=9D1DB10E-AFDA-4B4A-B796-F4CA359EFCFF&Options=&Search=) is for the PUD for 711 Church.. This would rezone 1.06 acres from R4C to a PUD that would allow a 12-17 story LEED Gold building with 273 units. It would also make a $7M contribution to the Affordable Housing Trust Fund.
+
+[C-2](https://a2gov.legistar.com/LegislationDetail.aspx?ID=6586223&GUID=BFA362E0-6389-4D99-8E01-574AC5AA1EE6&Options=&Search=) is the first reading of an ordinance that would make it so that the city does a Multimodal Transit Impact Analysis (MTIA) instead of a level of service analysis for new projects. This would expand transportation analyses so they're not only based on car traffic, but take into account pedestrian, transit, and cycling transportation as well.
+
+Finally we get to the resolutions. Tonight there are 4 economic development resolutions on the docket. [DC-1](https://a2gov.legistar.com/LegislationDetail.aspx?ID=6607006&GUID=A0C5AEA8-40A7-4AF9-8652-483B88135830&Options=&Search=) establishes an office of economic development and the administrator to "Prioritize Housing Development at all Income Levels to Support Housing Affordability, Sustainability, Tax Base Development, and Placemaking as the City's Strategic Priorities for Land Development." [DC-2](https://a2gov.legistar.com/LegislationDetail.aspx?ID=6607007&GUID=8848476B-C873-47D5-A029-B87B314B5248&Options=&Search=) is a resolution to direct the City Administrator to implement new processes and programs for housing development at all income levels to support housing affordability. [DC-3](https://a2gov.legistar.com/LegislationDetail.aspx?ID=6607008&GUID=E0B8DE58-F54E-4129-9E1D-0E56903BB2DB&Options=&Search=) directs the administrator to implement new processes and programs to support sustainability. [DC-4](https://a2gov.legistar.com/LegislationDetail.aspx?ID=6607009&GUID=9399494A-53F8-43CB-BA68-A4B311B3947F&Options=&Search=) is a resolution to direct the city to implement new processes to support placemaking and tax base improvements.
```

Comment:
s/and the administrator/and tasks the administrator/ ??

Reply:
And it's unclear whether various references to _administrator_ refers to the new office's administrator or the City Administrator. So maybe use _City Administrator_ when appropriate?